doc/reference/reference_lua/net_box.rst

```diff
@@ -40,13 +40,21 @@ Most ``net.box`` methods accept the last ``{options}`` argument, which can be:
 * ``{timeout=...}``. For example, a method whose last argument is
   ``{timeout=1.5}`` will stop after 1.5 seconds on the local node, although this
   does not guarantee that execution will stop on the remote server node.
-* ``{buffer=...}``. For an example see :ref:`buffer module <buffer-module>`.
+
+* ``{buffer=...}``. For example, see :ref:`buffer module <buffer-module>`.
```

Comment:
```suggestion
* ``{buffer=...}``. For an example, see the :ref:`buffer module <buffer-module>`.
```
This means "for an example of this option", not just idiomatically "for example".